lib/rex.rb

```diff
@@ -94,6 +94,14 @@ module Rex
 require 'rex/sslscan/scanner'
 require 'rex/sslscan/result'
 
+# Cryptography
+require 'rex/crypto/aes256'
+require 'rex/crypto/rc4'
+require 'rex/crypto/chacha20'
```

Comment:
@gwillcox-r7 Looks like some unrelated code has snuck in from the rebase 👀 

If you're happy waiting I think Dean will be able to fix this up on Monday :+1:

Reply:
Ok no problem